src/main.rs

```diff
@@ -173,6 +175,23 @@ fn validate_byte_size(s: String) -> Result<(), String> {
         |_| format!("{:?} is not a byte size", s))
 }
 
+// Create a regular expression from a file pattern:
+// Change
+// 1. "." to "\\."
+// 2. "%" to "."
+// 3. "*" to ".*"
+fn regexp_from_filepat(s: &str) -> Regex {
+    let string = s.to_string().replace(".","\\.").replace("%",".").replace("*",".*");
```

Comment:
Other special characters (like `[]`) will need to be escaped.

Or we could use the `glob` crate's implementation of filename patterns (it has a `Pattern` class that doesn't do filesystem access).

Reply:
Didn't know glob crate. Makes sense to use it.